docs/changelogs/1.9.x/1.9.1.md

```diff
@@ -0,0 +1,137 @@
+* Removed `ENDTURN_WEATHER_FORM` and `allowedToChangeFormInWeather` in [#5171](https://github.com/rh-hideout/pokeemerald-expansion/pull/5171)
+
+## 🧬 General 🧬
+### Added
+* Move Relearner UI now displays move category by @kittenchilly in [#5081](https://github.com/rh-hideout/pokeemerald-expansion/pull/5081)
+### Fixed
+* Fixes wrong padding field in `SpeciesInfo` struct by @AlexOn1ine in [#5139](https://github.com/rh-hideout/pokeemerald-expansion/pull/5139)
+* Fixed specific tiles changing to PC tiles when using Box Link/Debug PC option by @cawtds in [#5141](https://github.com/rh-hideout/pokeemerald-expansion/pull/5141)
+
+## 🐉 Pokémon 🐉
+### Fixed
+* Fixed stray transparent pixels in Urshifu sprites by @hedara90 in [#5071](https://github.com/rh-hideout/pokeemerald-expansion/pull/5071)
+* Fixed `bufferspeciesname` not working for species IDs over 1023 by @SBird1337 in [#5088](https://github.com/rh-hideout/pokeemerald-expansion/pull/5088)
+* Fixed overworld Pokémon breaking for species IDs above 1535 by @hedara90, @mrgriffin and
+@SarnPoke in [#5179](https://github.com/rh-hideout/pokeemerald-expansion/pull/5179)
+* Fixed overworld palettes for multiple species by @hedara90 in [#5107](https://github.com/rh-hideout/pokeemerald-expansion/pull/5107)
+  * Dialga Origin (Normal and Shiny)
+  * Palkia Origin (Normal and Shiny)
+  * Giratina Origin (shiny palette fixed by @hedara90 in [#5108](https://github.com/rh-hideout/pokeemerald-expansion/pull/5108))
+  * Xerneas Neutral/Active (Normal and Shiny)
+  * Enamorus Incarnate/Therian (Normal and Shiny)
+* Fixed/added missing Pokémon sprites and palettes by @Cafeei in [#5126](https://github.com/rh-hideout/pokeemerald-expansion/pull/5126)
+  * Overworld:
+    * Shiny Sneasler, Morelul, Bounsweet, Bruxish, Guzzlord, Regieleki, Zacian, Zamazenta
+    * Hisuian Zorua sprite
+    * Shiny Summer Sawsbuck
+    * Shiny Galarian Yamask, Darumaka, Zigzagoon, Zapdos, Ponyta, Rapidash, Slowpoke, Farfetch'd, Weezing, Mr. Mime, Articuno, Moltres, Slowking, Stunfisk, Darmanitan
+    * Shiny Hisuian Sneasel, Qwilfish, Samurott,
+  * Battle sprites:
+    * Shiny Sneasler, Cursola, Pincurchin, Runerigus
+    * Shiny Galarian Yamask, Darumaka
+* Fixed Unown Overworld follower sprites by Sarn by @hedara90 in [#5146](https://github.com/rh-hideout/pokeemerald-expansion/pull/5146)
+
+## ⚔️ Battle General ⚔️
+### Changed
+* Set new animation particles by default to off by @AlexOn1ine in [#5161](https://github.com/rh-hideout/pokeemerald-expansion/pull/5161)
+### Fixed
+* Fixed speed ties by @mrgriffin in [#4780](https://github.com/rh-hideout/pokeemerald-expansion/pull/4780)
+  * Cleanup by @hedara90 in [#5092](https://github.com/rh-hideout/pokeemerald-expansion/pull/5092)
+* Fixed Defiant/Competitive not working if the battler enters the field with a Court Changed Sticky Web on its side of the field by @PhallenTree in [#5093](https://github.com/rh-hideout/pokeemerald-expansion/pull/5093)
```

Comment:
```suggestion
* Fixed Defiant/Competitive not working after the battler enters the field with a Court Changed Sticky Web on its side of the field by @PhallenTree in [#5093](https://github.com/rh-hideout/pokeemerald-expansion/pull/5093)
```
Might be a better explanation of the problem, Defiant/Competetive correctly didn't trigger, but further stat drops also (incorrectly) didn't trigger the ability.